dissect/target/plugins/os/windows/sru.py

```diff
@@ -223,6 +225,22 @@
     ],
 )
 
+SRURecord: TypeAlias = Union[
```

Comment:
```suggestion
SRURecord = Union[
```
`TypeAlias` unfortunately requires >=3.10, so this is fine IMO.

Reply:
Removed in [71cb7ce](https://github.com/fox-it/dissect.target/pull/472/commits/71cb7ce665adfff82ebdcd0bf606325fbb73b718)